package.json

```
@@ -1,53 +1,140 @@
 {
-  "name": "express-rate-limit",
-  "version": "5.5.1",
-  "description": "Basic IP rate-limiting middleware for Express. Use to limit repeated requests to public APIs and/or endpoints such as password reset.",
-  "homepage": "https://github.com/nfriedly/express-rate-limit",
-  "author": {
-    "name": "Nathan Friedly",
-    "url": "http://nfriedly.com/"
-  },
-  "repository": "nfriedly/express-rate-limit",
-  "license": "MIT",
-  "main": "lib/express-rate-limit.js",
-  "files": [
-    "lib/"
-  ],
-  "keywords": [
-    "express-rate-limit",
-    "express",
-    "rate",
-    "limit",
-    "ratelimit",
-    "rate-limit",
-    "middleware",
-    "ip",
-    "auth",
-    "authorization",
-    "security",
-    "brute",
-    "force",
-    "bruteforce",
-    "brute-force",
-    "attack"
-  ],
-  "devDependencies": {
-    "bluebird": "^3.7.2",
-    "eslint": "^7.32.0",
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-prettier": "^4.0.0",
-    "express": "^4.17.1",
-    "husky": "^7.0.2",
-    "mocha": "^9.1.2",
-    "prettier": "^2.4.1",
-    "pretty-quick": "^3.1.1",
-    "sinon": "^11.1.2",
-    "supertest": "^6.1.6"
-  },
-  "scripts": {
-    "lint": "eslint .",
-    "autofix": "npm run lint -- --fix",
-    "test": "npm run lint && mocha",
-    "precommit": "pretty-quick --staged"
-  }
+	"name": "express-rate-limit",
+	"version": "6.0.0",
+	"description": "Basic IP rate-limiting middleware for Express. Use to limit repeated requests to public APIs and/or endpoints such as password reset.",
+	"author": {
+		"name": "Nathan Friedly",
+		"url": "http://nfriedly.com/"
+	},
+	"license": "MIT",
+	"homepage": "https://github.com/nfriedly/express-rate-limit",
+	"repository": "nfriedly/express-rate-limit",
+	"keywords": [
+		"express-rate-limit",
+		"express",
+		"rate",
+		"limit",
+		"ratelimit",
+		"rate-limit",
+		"middleware",
+		"ip",
+		"auth",
+		"authorization",
+		"security",
+		"brute",
+		"force",
+		"bruteforce",
+		"brute-force",
+		"attack"
+	],
+	"module": "dist/esm/index.js",
+	"main": "dist/cjs/index.js",
+	"exports": {
+		".": {
+			"import": "./dist/esm/index.js",
+			"require": "./dist/cjs/index.js"
+		}
+	},
+	"types": "dist/index.d.ts",
+	"type": "module",
+	"files": [
+		"dist/",
+		"tsconfig.json",
+		"package.json",
+		"package-lock.json",
+		"readme.md",
+		"license.md",
+		"changelog.md"
+	],
```

`request.writeEnabled` was added in node 12.9 - https://nodejs.org/api/http.html#requestwritableended (see also https://docs.npmjs.com/cli/v6/configuring-npm/package-json#engines )

```suggestion
	],
	"engines" : { "node" : ">=12.9.0" },
```

---

We are using `response.writableEnded`, not `request.writeEnabled`

---

`response.writableEnded` was added in Node 12.9 too 😆